src/main/java/com/list/todo/services/TodoListService.java

```diff
@@ -68,7 +70,7 @@ public void deleteTodoList(Long todoListId, Long userId) {
         Optional<TodoList> todoList = todoListRepository.findById(todoListId);
 
         if (todoList.isPresent()) {
```

Comment:
please refactor so that `isPresent` is not involved consider adding consumer using `Optional::ifPresent` https://docs.oracle.com/javase/8/docs/api/java/util/Optional.html#ifPresent-java.util.function.Consumer-